src/command_modules/azure-cli-appservice/azure/cli/command_modules/appservice/custom.py

```diff
@@ -284,6 +285,7 @@ def enable_zip_deploy(cmd, resource_group_name, name, src, timeout=None, slot=No
     # Read file content
     with open(os.path.realpath(os.path.expanduser(src)), 'rb') as fs:
         zip_content = fs.read()
+        logger.warning("Starting Zip Deployment")
```

Comment:
Use lower case such as `Starting zip deployment`? This is to be consistent with the other occasions mentioning the same term.